framework/framework.go

```diff
@@ -103,9 +103,15 @@ func (f *Framework) AddFunc(obj interface{}) {
 		}
 	}
 
+	rs, err := f.resourceRouter(ctx, obj)
```

Comment:
Is this like a convention thing? `rs` for `resources`and `r`for `resource`? I find this naming a bit confusing from the outside. A nitpick would be to use more verbal names.

Reply:
That is common in go.